recipes/jsonpath-rw/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "jsonpath-rw" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jsonpath-rw-{{ version }}.tar.gz
+  sha256: 05c471281c45ae113f6103d1268ec7a4831a2e96aa80de45edc89b11fac4fbec
+
+build:
+  entry_points:
+    - jsonpath.py = jsonpath_rw.bin.jsonpath:entry_point
```

Comment:
Windows is failing with:

```
[noarch_python] Noarch package contains binary script: jsonpath.exe
```

I guess that this entrypoint has multiple values? Maybe try adding

```
jsonpath = jsonpath_rw.bin.jsonpath:entry_point
```

too?